crates/holochain_zome_types/CHANGELOG.md

```diff
@@ -7,6 +7,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
 ## \[Unreleased\]
 
+- Reverts the below change so that `name` in DnaDef does effect on the DNA hash. Holochain does not support the DNA hash changing
+  so this change will remain in `0.2` but be removed from the `0.1` stream. That means that upgrading from `0.1` to `0.2` will be a breaking
+  change.
```

Comment:
It's a breaking change regardless of this revert.
```suggestion
  so this change will remain in `0.2` but be removed from the `0.1` stream.
  change.
```

Reply:
Yes, true, applied the suggested change